[2025-06-27T00:52:54.941+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T00:52:54.963+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-24T05:00:00+00:00 [queued]>
[2025-06-27T00:52:54.969+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-24T05:00:00+00:00 [queued]>
[2025-06-27T00:52:54.970+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T00:52:54.982+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-24 05:00:00+00:00
[2025-06-27T00:52:54.995+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=78) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T00:52:54.997+0000] {standard_task_runner.py:72} INFO - Started process 100 to run task
[2025-06-27T00:52:55.005+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt', 'create_ga4_events_dataset', 'scheduled__2025-06-24T05:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpqrf749ch']
[2025-06-27T00:52:55.016+0000] {standard_task_runner.py:105} INFO - Job 13: Subtask create_ga4_events_dataset
[2025-06-27T00:52:55.142+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-24T05:00:00+00:00 [running]> on host 9cd2c6ca0862
[2025-06-27T00:52:55.223+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-24T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-24T05:00:00+00:00'
[2025-06-27T00:52:55.230+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T00:52:55.239+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T00:52:55.260+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T00:52:55.268+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events.
[2025-06-27T00:52:55.269+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T00:52:55.271+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events in project: sixth-topic-349709 
[2025-06-27T00:52:56.428+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T00:52:56.493+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T00:52:56.497+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-24T05:00:00+00:00, execution_date=20250624T050000, start_date=20250627T005254, end_date=20250627T005256
[2025-06-27T00:52:56.606+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T00:52:56.632+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-06-27T02:50:51.884+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T02:50:51.895+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-24T05:00:00+00:00 [queued]>
[2025-06-27T02:50:51.901+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-24T05:00:00+00:00 [queued]>
[2025-06-27T02:50:51.901+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T02:50:51.909+0000] {taskinstance.py:2888} INFO - Executing <Task(BigQueryCreateEmptyDatasetOperator): create_ga4_events_dataset> on 2025-06-24 05:00:00+00:00
[2025-06-27T02:50:51.913+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=76) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T02:50:51.914+0000] {standard_task_runner.py:72} INFO - Started process 90 to run task
[2025-06-27T02:50:51.915+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt', 'create_ga4_events_dataset', 'scheduled__2025-06-24T05:00:00+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmplg6efsk_']
[2025-06-27T02:50:51.917+0000] {standard_task_runner.py:105} INFO - Job 86: Subtask create_ga4_events_dataset
[2025-06-27T02:50:51.952+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt.create_ga4_events_dataset scheduled__2025-06-24T05:00:00+00:00 [running]> on host ba60ab6371e3
[2025-06-27T02:50:51.998+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt' AIRFLOW_CTX_TASK_ID='create_ga4_events_dataset' AIRFLOW_CTX_EXECUTION_DATE='2025-06-24T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-24T05:00:00+00:00'
[2025-06-27T02:50:52.000+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T02:50:52.009+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-06-27T02:50:52.010+0000] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2025-06-27T02:50:52.011+0000] {bigquery.py:533} INFO - datasetId was not specified in `dataset_reference`. Will use default value ga4_events.
[2025-06-27T02:50:52.011+0000] {bigquery.py:533} INFO - projectId was not specified in `dataset_reference`. Will use default value sixth-topic-349709.
[2025-06-27T02:50:52.011+0000] {bigquery.py:544} INFO - Creating dataset: ga4_events in project: sixth-topic-349709 
[2025-06-27T02:50:53.169+0000] {bigquery.py:548} INFO - Dataset created successfully.
[2025-06-27T02:50:53.208+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T02:50:53.208+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt, task_id=create_ga4_events_dataset, run_id=scheduled__2025-06-24T05:00:00+00:00, execution_date=20250624T050000, start_date=20250627T025051, end_date=20250627T025053
[2025-06-27T02:50:53.235+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T02:50:53.255+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-27T02:50:53.256+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
